singularity-eos/closure/mixed_cell_models.hpp

```diff
@@ -170,9 +170,9 @@ class PTESolverBase {
   PORTABLE_INLINE_FUNCTION
   virtual void Fixup() const {
     Real vsum = 0;
-    for (int m = 0; m < nmat; ++m)
+    for (int m = 0; m < nmat; ++m) {
       vsum += vfrac[m];
-    PORTABLE_REQUIRE(vsum > 0., "Volume fraction sum is non-positive");
```

Comment:
Basically I found that the check could be sufficiently removed from the source of the issue that all it was telling me was that _something_ went wrong in the PTE solver. Instead of enforcing a check here, I decided to rely on the checks I have in the `get_sg_eos()` interface. See this comment (https://github.com/lanl/singularity-eos/pull/383#issuecomment-2223535810) for more discussion.

Granted, this isn't ideal since the `get_sg_eos()` interface is only used by one host code, but that's really just because of the use case I was looking at. I created an issue to move this code into a more generic form later that can be invoked by any host code: https://github.com/lanl/singularity-eos/issues/388